homeassistant/components/netgear/sensor.py

```diff
@@ -431,6 +431,8 @@ async def async_added_to_hass(self) -> None:
             sensor_data = await self.async_get_last_sensor_data()
             if sensor_data is not None:
                 self._value = sensor_data.native_value
+            else:
+                self.schedule_update_ha_state()
```

Comment:
The entity state is always updated by default when the entity is added to Home Assistant. We don't need this line.

What was the goal with this change?

Reply:
Actually in this case the update coordinator does not fetch new data from the API when the entity is added to prevent large bandwidth usage of speedtest during boot of homeassistant.

So the goal is to have the update coordinator fetch data from the API only if no previous values are available (so this is a new entity).

@MartinHjelmare did I use the wrong command?
I think so...
Schould I have used `await self.coordinator.async_request_refresh()`?

Reply:
Yes, that's correct. 👍